### DISCUSSION TITLE
Should winapps automatically configure the host?

### DISCUSSION THREAD

**Discussion by fbartels** [winapps-org/winapps#16] [Ideas]
I think there was a discussion about this in then old project, but I'm too lazy to search at the moment.

What do you think. Should winapps somehow configure the windows VM or install additional applications?

In https://github.com/winapps-org/winapps/discussions/9 there was a discussion already about using/recommending a specific base image. But should we use something like https://scoop.sh/ to also install applications?

**Comment by oskardotglobal** [comment]
Ideally we have a script so that the user just connects to the VM and runs it
That script installs the reg file and maybe more software like sunshine, but there's no real need to install software (besides maybe sunshine for gaming or more intense work)

**Reply by LDprg** [reply]
I think we could support remote Software install (for example with scoop) , but those package managers do not provide a method to install all software, for example Photoshop. It has to be installed manually. So I dont think it is worth the effort now.

Maybe we can add winget support as it is installed by default on windows. Still it doesn't support installing some common apps.

**Reply by oskardotglobal** [reply]
I guess, but its not really a priority
